packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeButtons.tsx

```diff
@@ -210,33 +212,35 @@ export function View3DButton(props: ModelsTreeHeaderButtonProps) {
   }, [models3d, props.viewport]);
 
   return (
-    <Button
-      size={props.density === "enlarged" ? "large" : "small"}
-      styleType="borderless"
-      title={TreeWidget.translate("modelsTree.buttons.toggle3d.tooltip")}
-      onClick={() => {
-        props.onFeatureUsed?.("models-tree-view3d");
-        void toggleModels(models3d, is3dToggleActive, props.viewport);
-      }}
-      disabled={models3d.length === 0}
-      endIcon={is3dToggleActive ? <SvgVisibilityShow /> : <SvgVisibilityHide />}
-    >
-      {TreeWidget.translate("modelsTree.buttons.toggle3d.label")}
-    </Button>
+    <Tooltip content={TreeWidget.translate("modelsTree.buttons.toggle3d.tooltip")}>
+      <Button
+        size={props.density === "enlarged" ? "large" : "small"}
+        styleType="borderless"
+        title={TreeWidget.translate("modelsTree.buttons.toggle3d.tooltip")}
```

Comment:
Same as with checkbox `title` should not be used.

Reply:
removed